test/closestPair_test.js

```diff
@@ -0,0 +1,27 @@
+import { expect } from 'chai'
+import closestPair from '../src/closestPair'
+
+describe('closestPair()', function(){
+
+  it('should be a function', function(){
+    expect(closestPair).to.be.a('function')
+  })
+
```

Comment:
You took the time in you function to check if a value is a array or null might as well test that functionality.